# A Government  company claims that an average light bulb lasts 270 days. A researcher randomly selects 18 bulbs for testing. The sampled bulbs last an average of 260 days, with a standard deviation of 90 days. If the CEO's claim were true, what is the probability that 18 randomly selected bulbs would have an average life of no more than 260 days

# here H-not = Mean < 260
# here H-one = Mean > 260


In [1]:
# WE HAVE TO CONDUCT, ONE TAIL t-TEST AND right TAILED TEST

In [2]:
# IMPORT REQUIRED LIBRARIES
from scipy.stats import t
import numpy as np

In [4]:
1- t.cdf(270,17,loc = 260 ,scale = 90/np.sqrt(18))

0.32167253567098353

# here p value = 0.32 or approximately 32% 

# Here p value is greater than the significance level of 0.05 , So we dont have sufficient evidence to reject null hypothesis

# That is the average life of the bulbs last is less tahn 260 days